debian/changelog

```diff
@@ -0,0 +1,5 @@
+aeternity-node (1.1.0) unstable; urgency=medium
```

Comment:
In Debian/Ubuntu packaging context, upstream should be us. :) This is the initial version hard-coded in the changelog file when packaging was started. A changelog file must be present before using other packaging tools. On package building `dch` generates proper entries with the correct version from the VERSION file. The latest version in this file is always the one in the VERSION file in the current source tree. Check the Makefile rules.

There might be a way to avoid this initial dummy version, but I had no time to play with `dch` with non-existing changelog file.

Reply:
What about empty file, this information is useless anyway as we're not going to track the changelog. Users should be point to the docs/release-notes of a specific commit

Reply:
An empty file will not work, already tested. The changelog file is mandatory for package creation. This is the place, from where the version of the software is determined in packaging context. The file is already a hack. When generated, it will just contain a note with the path to the release notes file. 


Example:

```
aeternity-node (2.2.0) bionic; urgency=medium

  * Release notes are available in /usr/share/doc/aeternity-
    node/docs/release-notes/RELEASE-NOTES-2.2.0.md

 -- Aeternity Team <info@aeternity.com>  Thu, 14 Mar 2019 08:38:12 +0000

aeternity-node (1.1.0) unstable; urgency=medium

  * Initial Debianization

 -- Valkov Ivaylo <ivaylo.valkov@quanterall.com>  Tue, 18 Dec 2018 09:43:00 +0200

```

I am aware it will not be tracked, that is why the approach taken is to generate it automatically. The situation about it is documented. However there is no work-around without it. 

The only other option is generating binary only packages with a lot of custom scripts/Makefiles. It is hard to maintain and proven ineffective. I am not in favor of this approach.